agent/bench-scripts/pbench-dbench

```diff
@@ -21,7 +21,7 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 benchmark_rpm=$script_name
 benchmark="dbench"
 if [[ -z "$benchmark_bin" ]]; then
-	benchmark_bin=/usr/local/bin/$benchmark
+	benchmark_bin="$benchmark"
```

Comment:
I think we need to use the following instead:

```
    benchmark_bin="$(command -v ${benchmark})"
```

We want to make sure we record the exact path the command that was used in our metadata.

Reply:
That makes sense but this won't work for aliases. How about `which --skip-alias --skip-functions $benchmark`?

Reply:
Why do we need to support aliases?

To support aliases means that the script has to available as an alias on all remote hosts when invoking.  This gets to be complicated to explain.

Reply:
In addition to remote host execution, if an alias resolves to more than one program, `benchmark_bin` won't provide the right value that will get used properly in all cases.

Reply:
Let's see how this goes using `which` as posted.  I've proposed a patch to isolate the change to one file so that if we have a problem with it we won't end up with 5 copies to fix.